e2e/testgames/game/Dockerfile

```
@@ -5,24 +5,60 @@ FROM golang:1.22-bookworm AS build
-RUN cd e2e/testgames && go mod download
-RUN cd e2e/testgames/game && go build -v -o /go/bin/app
+# build the binary with debugging symbols
+WORKDIR /go/src/app/e2e/testgames/game
+RUN go build -v -o /go/bin/app
 
 
 ################################
-# Runtime Image
+# Runtime Image - Normal
 ################################
 FROM gcr.io/distroless/base-debian12 AS runtime
```

Hadolint

[warning] Always tag the version of an image explicitly

(DL3006)